wagtail_footnotes/static/footnotes/js/footnotes.js

```diff
@@ -47,19 +47,19 @@ class FootnoteSource extends React.Component {
         table.empty();
 
         var live_footnotes = document.querySelectorAll(
-          "#id_footnotes-FORMS > li:not(.deleted)"
+          "#id_footnotes-FORMS .w-panel"
```

Comment:
w-panel is only added in Wagtail 4, so won't work in 2.15

Reply:
🤦 Oh of course. I was so pleased I'd fixed 4 🥲